### PR TITLE
Edits Account Generation Page

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

**Review by jsidorenko — APPROVED** [review]
Looks much better!

File: docs/learn/learn-account-generation.md [hunk 1]
```diff
@@ -32,128 +27,86 @@ for more information about key security.
...
+:::info Treasury-funded Wallets and Browser Extensions
 
-:::info local in-browser account storage is disabled by default on the Polkadot-JS UI
-
-To create an account using the Polkadot-JS UI, navigate to settings > account options and click on
-allow local in-browser account storage in the drop-down menu.
+You can create accounts using treasury-funded
+[wallets and extensions](../general/wallets-and-extensions.md). Bear in mind that some of these are
+**unaudited** and are not officially affiliated with Web3 Foundation or the
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} project unless otherwise stated.
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]:
```suggestion
If you are a general user, you are recommended to create accounts using any of the
[wallets and extensions](../general/wallets-and-extensions.md) funded by the Treasury or Grants. If you are a power user or a developer, you can use [Polkadot-JS Extension](../polkadotjs.md#polkadot-js-extension).
```

File: docs/learn/learn-account-generation.md [hunk 2]
```diff
@@ -32,128 +27,86 @@ for more information about key security.
...
-
-Please note that as this keeps your accounts in the browser extension, it is unsafe to hold
-significant funds on them. By definition, a browser is a "hot wallet" and is susceptible to a wide
-range of attacks, so keep your funds in cold storage when dealing with non-trivial amounts. For
-improved security, you securely stash away the seed phrase for your accounts and remove all traces
-of the accounts from your computer after creating them.
-
-Besides the extension and the default UI,
+You can also create accounts using
+[the Polkadot-JS browser extension](#polkadot-js-browser-extension). This extension remembers your
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]:
```suggestion
When you create accounts using browser extensions like
[the Polkadot-JS extension](#polkadot-js-browser-extension), it stores your
```

File: docs/learn/learn-account-generation.md [hunk 3]
```diff
@@ -32,128 +27,86 @@ for more information about key security.
...
+accounts and lets you clear your browser cache without fear. Remember to back up your seed phrase -
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]:
```suggestion
accounts in a private vault and lets you clear your browser cache without fear. Remember to back up your seed phrase -
```

File: docs/learn/learn-account-generation.md [hunk 4]
```diff
@@ -32,128 +27,86 @@ for more information about key security.
...
+if you lose access to this computer or the extension somehow crashes beyond repair, the seed phrase
+will come in handy. Alternatively, you can create accounts using the
+[Polkadot-JS UI](#polkadot-js-ui), although this option is disabled by default. More info about the
+Polkadot-JS UI can be found on [the dedicated page](../general/polkadotjs-ui.md).
+
+Please note that as this keeps your accounts in the browser extension, holding significant funds on
+them is unsafe. By definition, a browser is a "hot wallet" susceptible to a wide range of attacks,
+so keep your funds in cold storage when dealing with non-trivial amounts.
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]:
```suggestion
Please note that all wallets that are connected to the internet are considered "hot wallets" that are susceptible to a wide range of attacks,
so it is recommended to use cold storage when dealing with non-trivial amounts.
```

File: docs/learn/learn-account-generation.md [hunk 5]
```diff
@@ -32,128 +27,86 @@ for more information about key security.
...
+
+Current popular cold storage solutions within the
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} ecosystem include
+[Ledger](../general/ledger.md) and [Polkadot Vault](../general/polkadot-vault.md). The latter keeps
+your keys on an air-gapped mobile phone. However, it does require obtaining an old Android or
+iOS-compatible phone that you are comfortable using only for Polkadot Vault.
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]:
```suggestion
your keys on an air-gapped mobile phone. However, it does require obtaining a dedicated Android or
iOS-compatible phone that you are comfortable using only for Polkadot Vault.
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**